crates/router/src/core/payments/operations/payment_update.rs

```diff
@@ -82,6 +82,7 @@ impl<F: Send + Clone> GetTracker<F, PaymentData<F>, api::PaymentsRequest> for Pa
             )
             .await?;
 
+        //is this really needed? payment_intent is already fetched at line 55
```

Comment:
Please avoid mentioning line numbers in code, they'll be invalidated as code changes.

Reply:
This can be removed.